spec/factories.rb

```diff
@@ -0,0 +1,33 @@
+FactoryGirl.define do
+  sequence :lat do |n|
+    "#{n}.0".to_f
+  end
+
+  sequence :lon do |n|
+    "#{n}.0".to_f
```

Comment:
When I use those for rand I get a diff of 
 -"lon" => 7.210242433246464,
 +"lon" => 7.21024243324646,